internal/scm/github/github.go

```diff
@@ -187,9 +197,36 @@ func (g *Github) GetRepositories(ctx context.Context) ([]scm.Repository, error)
 	return repos, nil
 }
 
+func (g *Github) excludeRepositoryFilter(repoName string) (bool, error) {
+	if g.RepositoryExcludeFilter.Err != nil {
+		fmt.Println("repo-exclude RegEx Error: ", g.RepositoryExcludeFilter.Err)
```

Comment:
If you want to log, use the logrus logger. But in this case, an error in the regexp should have caused the run to halt before it started.